bucket/tailspin.json

```
@@ -0,0 +1,22 @@
+{
+    "version": "2.2.0",
+    "description": "🌀 A log file highlighter",
+    "homepage": "https://github.com/bensadeh/tailspin",
+    "license": "MIT",
+    "depends": "main/less",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/bensadeh/tailspin/releases/download/2.2.0/tspin-x86_64-pc-windows-msvc.zip",
+            "hash": "8c240faddb76b1457be541612d39ff242f71ca359d1e447280e38f8a71129b30"
+        }
+    },
+    "bin": "tspin.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/bensadeh/tailspin/releases/download/$version/tspin-x86_64-pc-windows-msvc.zip"

```

```suggestion
                "url": "https://github.com/bensadeh/tailspin/releases/download/$version/tailspin-x86_64-pc-windows-msvc.zip"

```

Archive name changed since 2.3.0.
Please update to the latest version after fixing autoupdate.